std_msgs/Header header
float64 targetZ               # targetZ reading
float64 targetZ_variance      # targetZ_variance reading


